### ISSUE TITLE
Help needed to validate statistical assumptions in metrics

### ISSUE THREAD

**Issue by DimitriFrischmann** [DimitriFrischmann/Regolith#31]
Documentation for the custom loss function in the TCR-sequencing module is outdated. The current implementation uses a different smoothing parameter than what is described.

**Comment by DimitriFrischmann** [comment]
Closes #32 We cannot fix the performance issue with extremely small batch sizes as it's an inherent limitation of the underlying distributed training framework. The overhead of coordination between workers outweighs benefits at very small batch sizes.
